### DISCUSSION TITLE
Accessing the HTTP Stream from Restreamer

### DISCUSSION THREAD

**Discussion by 7ekhed** [datarhei/restreamer#602] [Q&A] [answered]
Hello Restreamer Community!

Still in love with this product, and have a somewhat abstract usecase for it

I'm trying to grab the HTTP URL of a stream, like, when on the stream viewer you can view the stream like a video. What i'm trying to do is grab that HTTP URL and import it into another application, i'm not using OBS for this but OBS as an example, I want to be able to grab the HTTP stream from the viewer page, and import it into like a Media Plaer element on OBS, though I can't seem to figure out how to grab it

When F12-ing, I see a blob HTTP link, but there doesn't seem to be a way for me to just load the raw stream on an HTTP

Are you all aware of how I can do this?

Edit: HLS doesn't seem to be quite right, it doesn't let me open a new browser tab and view, I feel as though there should be a way to natively open an HTTP URL in a browser even and view the stream

Thank you!

**Comment by svenerbeck** [comment, marked as answer]
Hello 7ekhed,

We're pleased to hear that you value our product! Regarding your query about the HTTP URL of the stream:

- M3U8 (HLS): You can use the M3U8 address (HLS) for your stream. You'll find the button for this in the dashboard. Click on "HLS", and the respective address will be displayed for you. This M3U8 URL can be used directly in many applications and players.
- RTMP and SRT: Alternatively, you can utilize the RTMP or SRT address, depending on which best suits your use case.

Please remember to activate these modules in the settings before using them.

I hope this helps. Should you have any more questions or concerns, please don't hesitate to reach out!

Cheerio,
Sven
